Graphiql enhancements (#19371)
backstage/backstage: .changeset/heavy-toys-pull.md

---

> '@backstage/catalog-model': minor

Reviewer: the examples aren't shipped with the package - remove this line

'@backstage/plugin-api-docs': minor
'@backstage/plugin-graphiql': minor
---

> Enhanced API-docs for graphql to enable user try out the api if they provide backstage.io/api-graphql-url annotation in catalog entity.

Reviewer: ```suggestion
Enhanced the `api-docs` plugin for graphql to enable users to try out the api if they provide a `backstage.io/api-graphql-url` annotation in their catalog entity.
```
